rust_src/src/frames.rs

```diff
@@ -181,7 +181,7 @@ pub fn set_frame_selected_window(
         error!("In `set-frame-selected-window', WINDOW is not on FRAME")
     }
     if frame_ref == selected_frame().as_frame().unwrap() {
-        unsafe { Fselect_window(window, norecord) }
+        select_window(window, norecord)
```

Comment:
This should be a call to `select_window_lisp`.

Reply:
@shaleh You prefer that stylistically over the `select_window_lisp as select_window` alias in the use declaration?  I can dig that.  I'll update the declaration.

Reply:
Code is meant to be read as much as it is executed. Imagine someone 30 years from now porting this from Rust to AmazeBalls3000 (the preferred language of the next generation). They will be scratching their head over the multiple `select_window`s.